src/theory/uf/theory_uf_rewriter.cpp

```diff
@@ -204,6 +210,55 @@ Node TheoryUfRewriter::decomposeHoApply(TNode n,
 }
 bool TheoryUfRewriter::canUseAsApplyUfOperator(TNode n) { return n.isVar(); }
 
+Node TheoryUfRewriter::rewriteLambda(Node node)
+{
+  // The following code ensures that if node is equivalent to a constant
```

Comment:
To prevent calling this function with the wrong kind,
the following assertion can be added here `Assert(node.getKind() == kind::LAMBDA);`.